release-please-config.json

```diff
@@ -0,0 +1,160 @@
+{
+    "packages": {
+        ".": {
+            "last-release-sha": "da19158e7a02c574d4f6d3d367ee264cb08d47ec",
+            "release-type": "node",
+            "changelog-sections": [
+                {
+                    "type": "feat",
+                    "section": "Features",
+                    "hidden": false
+                },
```

Comment:
Just a style/indentation thing, but:

I would personally prefer each of these to be on one line like:

```json
                { "type": "feat", "section": "Features", "hidden": false },
```

So the file is easier to skim over quickly, but I'm not sure what other contributors to this repo think about it? (But IMO a subjective style thing should _not_ be a blocker for merging this. It's not quite that important either way.)